cmds/kexec/kexec.go

```diff
@@ -5,95 +5,111 @@
 // kexec executes a new kernel over the running kernel (u-root).
 //
 // Synopsis:
-//     kexec [--dryrun] [--cmdline=] [--i=] [FILES...]
+//     kexec [--initrd=FILE] [--command-line=STRING] [-l] [-e] [KERNELIMAGE]
 //
 // Description:
-//     This is only intended to be used with kexec_load_files, not the older
-//     kexec.
+//		 Loads a kernel for later execution.
 //
 // Options:
-//     -dryrun:   do not do kexec system calls
-//     --cmdline: command line for kernel
-//     --i:       initramfs
+//     --cmdline=STRING:       command line for kernel
```

Comment:
personally, I think the single letter options are fine. Do we really need the long ones?

Reply:
we do to be compatible with kexec(8).